scripts/esbuild/build.ts

```diff
@@ -8,14 +10,32 @@ import { buildScreenshot } from './screenshot';
 import { buildSysNode } from './sys-node';
 import { buildTesting } from './testing';
 
-// the main entry point for the Esbuild-based build
-async function main() {
+// the main entry point for the build
+export async function run(rootDir: string, args: ReadonlyArray<string>) {
   const opts = getOptions(process.cwd(), {
-    isProd: !!process.argv.includes('--prod'),
-    isCI: !!process.argv.includes('--ci'),
-    isWatch: !!process.argv.includes('--watch'),
+    isProd: !!args.includes('--prod'),
+    isCI: !!args.includes('--ci'),
+    isWatch: !!args.includes('--watch'),
```

Comment:
Non blocking - IDK if we need the double bang args here - shouldn't `includes` return a boolean?

Reply:
haha yeah we definitely don't need those - looks like I added them in #4187 when `scripts/esbuild/build.ts` was added but I don't remember why! going to remove